packages/design-system/src/components/web-components/ds-tabs/ds-tabs.stories.tsx

```diff
@@ -13,8 +12,27 @@ const meta: Meta = {
     docs: {
       page: WebComponentDocTemplate,
       description: {
-        component:
-          'For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/tabs/).',
+        component: `
+For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/tabs/).
+
+The \`ds-tabs\` component accepts regular \`<div>\` elements as children to represent tab panels. Each \`<div>\` should contain specific content and attributes for tab functionality.
```

Comment:
Once we have markdown capabilities in storybook `argType` descriptions ([pr-3303](https://github.com/CMSgov/design-system/pull/3303)) we might consider relocating this documentation.